index.html

```diff
@@ -1,9 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-  </head>
-  <body></body>
+</head>
+<body>
+    <!-- <p>git add .   // adds the files that should be commited
+       
+       
+         git commit -m “Message” // creates the commit with the files added
+        
+        
+         git push // upload the files in the cloud</p> -->
+
+
+         <!-- git checkout -b “branch name” -------- create a brench -->
+       
+         <!-- git branch // see the branch where you are -->
```

Comment:
You should remove those comments as they are not used in the code.

Reply:
Ok